.github/workflows/e2e-wasm.yml

```
@@ -0,0 +1,43 @@
+---
+name: CosmWasm e2e contract tests
+
+on:
+  # On normal PRs or when workflow goreleaser finishes, as it gets the last release tag.
+  pull_request:
+    paths: ["**.go", "**.proto", "go.mod", "go.sum"]
+  workflow_run:
+    workflows: ["goreleaser"]
+    types:
+      - completed
+
```

```suggestion
  pull_request:
    paths: ["**.go", "**.proto", "go.mod", "go.sum"]
  workflow_run:
    workflows: ["goreleaser"]
    types:
      - completed
  workflow_dispatch:
```

Adding the `workflow_dispatch` condition will allow us to manually run this and test against the same release multiple times.